src/explore-education-statistics-admin/src/pages/theme/topic/CreatePublicationPage.tsx

```diff
@@ -144,7 +144,10 @@ const CreatePublicationPage = ({
               contact => contact.contactName,
             )[0].id,
             methodologyChoice: 'existing',
-            selectedMethodologyId: '',
+            selectedMethodologyId: orderBy(
```

Comment:
I'm surprised we automatically try and select the first methodology for a publication. Typically you would want the user to actually think about it before accidentally submitting the form with the wrong thing selected.

Fine to approve this though as it does solve the ticket.